CONTRIBUTING.md

```diff
@@ -18,45 +18,163 @@ any additional information for contributing code to this project.
 
 ## Building
 
-### Egg format
+### Static/portable CLI binary
 
 ```
-$ ./bin/build
+$ ./bin/build_binary
 ```
 
-### Static/portable CLI binary
+### Egg format
```

Comment:
what's `Egg format`?

Reply:
egg contains all the source files for a Python compressed so basically ZIP files that you can import into the python program without unzipping